src/main/java/com/google/sps/servlets/SessionInformationServlet.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some information about the session. */
+@WebServlet("/get-session-info")
```

Comment:
Is this servlet going to return anything *other* than a Session object? If not, just call it get-session. And rename some of the other stuff too